src/index.html

```diff
@@ -145,5 +150,48 @@ <h2 class="modal__title">New post</h2>
         <p class="modal__caption"></p>
       </div>
     </div>
+
+    <div class="modal" id="edit-avatar-modal">
+      <div class="modal__container">
+        <button type="button" class="modal__close-btn"></button>
+        <h2 class="modal__title">Edit profile avatar</h2>
+        <form
+          name="edit-avatar-profile"
+          class="modal__form"
+          id="profile-avatar-form"
+          novalidate
+        >
+          <label for="profile-avatar-input" class="modal__label">
+            Profile Picture
+            <input
+              id="profile-avatar-input"
+              type="text"
```

Comment:
`  type="text"`

* Please, add `type="url"` for the `avatar` input, to validate it as a `link`